browser-test/src/support/admin_programs.ts

```diff
@@ -736,8 +771,13 @@ export class AdminPrograms {
     programName: string,
     blockDescription = 'screen description',
     questions: QuestionSpec[] = [],
+    isProgramdisabled: boolean = false,
```

Comment:
nitpick: I found another one that should be `isProgramDisabled`. This might be it, but please check if there are others.

Reply:
change applied. also scanned through the files to avoid similar typo.